query_execution/PolicyEnforcerBase.cpp

```diff
@@ -60,6 +61,23 @@ void PolicyEnforcerBase::processMessage(const TaggedMessage &tagged_message) {
       admitted_queries_[query_id]->processWorkOrderCompleteMessage(op_index);
       break;
     }
+#ifdef QUICKSTEP_DISTRIBUTED
```

Comment:
Why should we use this macro in the base class? Shouldn't this piece code go to the distributed derived class of the PolicyEnforcer?